swyft/store/store.py

```diff
@@ -445,7 +458,6 @@ def simulate(
         # Run simulations and collect status
         if len(idx) == 0:
             log.debug("No simulations required.")
-            return
```

Comment:
Cannot return early here, if one has set `wait_for_simulations=True`, needs to go through the final if statement in the method.